src/message_interface/response.rs

```diff
@@ -35,15 +38,27 @@ pub enum Response {
     /// GetOutputsWithAdditionalUnlockConditions)
     OutputIds(Vec<OutputId>),
     /// Response for [`GetOutput`](crate::message_interface::AccountMethod::GetOutput)
-    Output(Box<Option<OutputData>>),
+    Output(Box<Option<OutputDataDto>>),
```

Comment:
What about `Option<BoxOutputDataDto>>` ? It would avoid a dynamic allocation in the case of None